src/ui/server/controllers/insurance/policy/map-and-save/nominated-loss-payee/index.ts

```diff
@@ -0,0 +1,44 @@
+import hasFormData from '../../../../../helpers/has-form-data';
+import mapSubmittedData from '../../map-submitted-data/jointly-insured-party';
+import save from '../../save-data/nominated-loss-payee';
+import { Application, RequestBody, ValidationErrors } from '../../../../../../types';
+
+/**
+ * mapAndSave nominatedLossPayee
+ * Map and save any valid nominatedLossPayee fields
+ * @param {Express.Request.body} Express request body
+ * @param {Application}
+ * @param {Object} Validation errors
+ * @returns {Boolean}
+ */
+const nominatedLossPayee = async (formBody: RequestBody, application: Application, validationErrors?: ValidationErrors) => {
+  try {
+    if (hasFormData(formBody)) {
+      const populatedData = mapSubmittedData(formBody);
+
+      let saveResponse;
+
+      if (validationErrors) {
+        saveResponse = await save.nominatedLossPayee(application, populatedData, validationErrors.errorList);
+      } else {
+        saveResponse = await save.nominatedLossPayee(application, populatedData);
+      }
+
+      if (!saveResponse) {
+        return false;
+      }
+
+      return true;
+    }
+
+    return true;
+  } catch (err) {
+    console.error('Error mapping and saving application - policy - jointly insured party %O', err);
```

Comment:
wrong console.error?

Reply:
Will fix in next PR along with some other typos